robot.py

```diff
@@ -103,7 +103,7 @@ async def _wiggle_on_inactivity(self):
             while True:
                 await asyncio.sleep(self.INACTIVITY_PERIOD_S)
                 self._logger.debug("wiggle wiggle wiggle")
-                for _ in range(3):
+                for _ in range(1):
                     await self._servo.move(self.UPPER_POSITION +
                                            self.WIGGLE_AMOUNT)
                     await asyncio.sleep(self.WIGGLE_DELAY_S)
```

Comment:
I can't comment on line 113, but I'd say "wiggling" instead of "wiggle." While we're changing this file.

Reply:
Done!